pkg/codec/vaapi/vp8.go

```diff
@@ -545,6 +547,10 @@ func (e *encoderVP8) Controller() codec.EncoderController {
 	return e
 }
 
+func (e *encoderVP8) ForceKeyFrame() {
+	e.forceKeyFrame = true
```

Comment:
It may cause data race if `ForceKeyFrame()` is called from other goroutine during `Read()`?

Reply:
I've changed the type to use an `atomic.Bool` instead. This way, there is no data race possible.